### ISSUE TITLE
`plot.infl.rma.uni()` changes the `mfrow` option of `par` but does not change it back if `plotinf=FALSE` and `plotdfbs=TRUE`

### ISSUE THREAD

**Issue by sfcheung** [wviechtb/metafor#50]
## Classification:

Bug Report

## Summary

`plot.infl.rma.uni()` changes the `mfrow` option of `par` but does not change it back if `plotinf=FALSE` and  `plotdfbs=TRUE`.

## Reproducible Example (if applicable)

Adapted from the example of `plot.infl.rma.uni()`

```r
dat <- escalc(measure="RR", ai=tpos, bi=tneg, ci=cpos, di=cneg, data=dat.bcg)
res <- rma(yi, vi, mods = ~ ablat + year, data=dat)
inf <- influence(res)
plot(inf, plotinf=FALSE, plotdfbs=TRUE)
plot(rnorm(10), rnorm(10))
```

The last plot will be plotted on a 3-by-1 layout created by `plot.infl.rma.uni()`.

## Notes

If `plotinf = TRUE`, then these lines will run, whether `plotdfbs` is `TRUE` or not.

https://github.com/wviechtb/metafor/blob/f0a5607b4798f98539685abd2dc377e2df84eda1/R/plot.infl.rma.uni.r#L93-L94

However, if `plotinf = FALSE`, then the lines above will not be run. `mfrow` will be changed when `dfbs` graphs are plotted, but the original value will not be restored.

How about moving the lines 93-94 before the `if (plotinf)` block, such that the `mfrow` value is stored and restored in all conditions?

## sessionInfo()

```
R version 4.1.1 (2021-08-10)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=Chinese (Traditional)_Hong Kong SAR.950  LC_CTYPE=Chinese (Traditional)_Hong Kong SAR.950   
[3] LC_MONETARY=Chinese (Traditional)_Hong Kong SAR.950 LC_NUMERIC=C                                       
[5] LC_TIME=Chinese (Traditional)_Hong Kong SAR.950    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] metafor_3.0-2 Matrix_1.3-4 

loaded via a namespace (and not attached):
[1] compiler_4.1.1  tools_4.1.1     mathjaxr_1.4-0  nlme_3.1-152    grid_4.1.1      lattice_0.20-44
```

**Comment by wviechtb** [comment]
Thanks for raising this issue. Will take a closer look asap.

**Comment by wviechtb** [comment]
Fixed with ae76b02.
